ramips: cleaning up nvmem-cells definitions

namiltd opened the pull request:
HiWiFi HC5861 - inside &ethernet - removed due to duplicate in mt7620a_hiwifi_hc5x61.dtsi
TP-Link Archer C2 v1 - inside &ethernet - removed due to duplication in mt7620a_tplink_8m.dtsi
Xiaomi Mi AC1200 WLAN Range Extender RA75 - inside &wmac - removed due to duplication in mt7628an_xiaomi_mi router-4.dtsi

CUDY X6 v1,
CUDY X6 v2 - inside &wifi - moved from mt7621_cudy_x6.dtsi

ELECOM WMC-M1267GST2,
ELECOM WMC-S1267GS2,
ELECOM WRC-1167GS2-B,
ELECOM WRC-1167GST2 - inside &wifi - eeprom moved from mt7621_elecom_wrc-gs-1pci.dtsi

Netgear R6020,
Netgear R6080,
Netgear R6120 - inside &wifi5 - moved from mt7628an_netgear_r6xxx.dtsi

TP-Link Archer C20 v4,
TP-Link Archer C50 v3,
TP-Link TL-MR3420 v5,
TP-Link TL-WA801ND v5,
TP-Link TL-WR840N v4,
TP-Link TL-WR841N v13,
TP-Link TL-WR842N v5,
TP-Link TL-WR850N v2,
TP-Link TL-WR902AC v3,
TP-Link TL-WR902AC v4 - inside &ethernet - moved from mt7628an_tplink_8m.dtsi 
(for compatibility with other files in which mt7628an_tplink_8m.dtsi is loaded, to avoid overwriting)

DragonBluep commented on target/linux/ramips/dts/mt7628an_netgear_r6xxx.dtsi:
```diff
@@ -138,8 +138,6 @@
 	wifi5: wifi@0,0 {
 		reg = <0x0000 0 0 0 0>;
-		nvmem-cells = <&eeprom_factory_8000>;
-		nvmem-cell-names = "eeprom";
```
Missing eeprom from `Netgear R6120`.

namiltd replied:
Fixed

mpratt14 commented:
please split the commit up into multiple commits based on the model, or group of models in the same series if there is a shared DTSI (or at least by brand)

you can refer to #14666 to see what I mean

if you need help with Git, feel free to ask any of us

namiltd committed:
ramips: hiwifi,hc5861: cleaning up nvmem-cells definitions

Remove nvmem-cells definitions due to duplication
in mt7620a_hiwifi_hc5x61.dtsi

Signed-off-by: Mieczyslaw Nalewaj <namiltd@yahoo.com>

namiltd committed:
ramips: tplink,archer-c2-v1: cleaning up nvmem-cells definitions

Remove nvmem-cells definitions due to duplication
in mt7620a_tplink_8m.dtsi

Signed-off-by: Mieczyslaw Nalewaj <namiltd@yahoo.com>

namiltd committed:
ramips: xiaomi,mi-ra75: cleaning up nvmem-cells definitions

Remove nvmem-cells definitions due to duplication
in mt7628an_xiaomi_mi-router-4.dtsi

Signed-off-by: Mieczyslaw Nalewaj <namiltd@yahoo.com>

namiltd committed:
ramips: cudy_x6: cleaning up nvmem-cells definitions

Move nvmem-cells definitions to dts files to prevent overwriting

Signed-off-by: Mieczyslaw Nalewaj <namiltd@yahoo.com>

namiltd committed:
ramips: elecom_wrc-gs-1pci: cleaning up nvmem-cells definitions

Move nvmem-cells definitions to dts files to prevent overwriting

Signed-off-by: Mieczyslaw Nalewaj <namiltd@yahoo.com>

namiltd committed:
ramips: netgear_r6xxx: cleaning up nvmem-cells definitions

Move nvmem-cells definitions to dts files to prevent overwriting

Signed-off-by: Mieczyslaw Nalewaj <namiltd@yahoo.com>

namiltd committed:
ramips: tplink_8m: cleaning up nvmem-cells definitions

Move nvmem-cells definitions to dts files for compatibility with other files
in which mt7628an_tplink_8m.dtsi is loaded, to prevent overwriting

Signed-off-by: Mieczyslaw Nalewaj <namiltd@yahoo.com>

namiltd commented:
@mpratt14 Done
